### ISSUE TITLE
Should the `trades` stream delegate to the same endpoint rather than `aggTrades?

### ISSUE THREAD

**Issue by dmitriz** [HyperCubeProject/binance-api-node#90]
It seems that the `ws.trades` method delegates to the `aggTrades` endpoint:

https://github.com/HyperCubeProject/binance-api-node/blob/master/src/websocket.js#L165-L167

instead of using the `trades` endpoint as indicated in the readme:

https://github.com/HyperCubeProject/binance-api-node#trades-1

Is this a bug?

UPDATE: fixed the last link to point to WS rather than REST

**Comment by balthazar** [comment]
The websocket and http client api is not the same

**Comment by dmitriz** [comment]
@balthazar 
Thanks, I have changed the Readme link now to this part from the websocket section:
https://github.com/HyperCubeProject/binance-api-node#trades-1

It is called "trades" but retrieves "aggTrades" instead:
```js
client.ws.trades(['ETHBTC', 'BNBBTC'], trade => {
  console.log(trade)
})

Output

{
  eventType: 'aggTrade',
  eventTime: 1508614495052,
  symbol: 'ETHBTC',
  price: '0.04923600',
  quantity: '3.43500000',
  maker: false,
  tradeId: 2148226
}
```

On the other hand, the [trades stream](https://github.com/binance-exchange/binance-official-api-docs/blob/master/web-socket-streams.md#trade-streams) from the official doc has 
```js
Stream Name: <symbol>@trade

Payload:

{
  "e": "trade",     // Event type
  "E": 123456789,   // Event time
  "s": "BNBBTC",    // Symbol
  "t": 12345,       // Trade ID
  "p": "0.001",     // Price
  "q": "100",       // Quantity
  "b": 88,          // Buyer order Id
  "a": 50,          // Seller order Id
  "T": 123456785,   // Trade time
  "m": true,        // Is the buyer the market maker?
  "M": true         // Ignore.
}
```

That stream does not seem to be accessible at all from this library. Is that intentional?

**Comment by balthazar** [comment]
Indeed it's not, I believe the trade steam was previously not exposed by the API, and I simply called `aggTrades` `trades`. Will need to implement it and rename the other one. Thanks for the heads-up!

**Comment by balthazar** [comment]
Released `0.8.0`.

**Comment by milkyklim** [comment]
Binance [documentation](https://github.com/binance-exchange/binance-official-api-docs/blob/master/web-socket-streams.md) says that `aggTrade` endpoint should have 
```
  "f": 100,         // First trade ID
  "l": 105,         // Last trade ID 
``` 
fields. 

But I don't get them returned when use `client.ws.aggTrades`. Is this a bug or something intentional?
